crates/re_types/definitions/rerun/archetypes/clear.fbs

```diff
@@ -15,5 +15,5 @@ table Clear (
   "attr.rust.derive": "PartialEq, Eq",
   order: 100
 ) {
-  settings: rerun.components.ClearSettings ("attr.rerun.component_required", order: 100);
+  settings: rerun.components.ClearIsRecursive ("attr.rerun.component_required", order: 100);
```

Comment:
```suggestion
  recursive: rerun.components.ClearIsRecursive ("attr.rerun.component_required", order: 100);
```

I think this should be named as recursive so that API looks like:
```
rr.log("path", rr.Clear(recursive=True))
```
instead of:
```
rr.log("path", rr.Clear(settings=True))
```

Reply:
Good point!